Surface_mesh_segmentation/examples/Surface_mesh_segmentation/sdf_values_example.cpp

```diff
@@ -17,7 +17,7 @@ int main()
   Polyhedron mesh;
   std::ifstream input("data/cactus.off");
   if ( !input || !(input >> mesh) || mesh.empty() || ( !CGAL::is_triangle_mesh(mesh)) ) {
-    std::cerr << "Input is not a triangle mesh" << std::endl;
+    std::cout << "Input is not a triangle mesh" << std::endl;
```

Comment:
If wonder what rule you are following. `std::cerr` corresponds to the "error output stream". That seems a good choice to display errors to.